src/server/pfs/fuse/options.go

```diff
@@ -35,3 +74,24 @@ func (o *Options) getUnmount() chan struct{} {
 	}
 	return o.Unmount
 }
+
+func (o *Options) validate(c *client.APIClient) error {
+	if o == nil {
+		return nil
+	}
+	for repo, opts := range o.RepoOptions {
+		if opts.Write {
+			if uuid.IsUUIDWithoutDashes(opts.Branch) {
+				return errors.Errorf("can't mount commit %s@%s in Write mode (mount a branch instead)", repo, opts.Branch)
+			}
+			bi, err := c.InspectBranch(repo, opts.Branch)
+			if err != nil && !errutil.IsNotFoundError(err) {
+				return err
+			}
+			if bi != nil && len(bi.Provenance) > 0 {
+				return errors.Errorf("can't mount branch %s@%s in Write mode because it's an output branch")
```

Comment:
missing the formatting parameters here
```suggestion
				return errors.Errorf("can't mount branch %s@%s in Write mode because it's an output branch", repo, opts.Branch)
```